linebot/models/mentionee.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License. You may obtain
+#  a copy of the License at
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+
+"""linebot.models.mentionee module."""
+
+from __future__ import unicode_literals
+
+from abc import ABCMeta
+
+from future.utils import with_metaclass
+
+from .base import Base
+
+
+class Mentionee(with_metaclass(ABCMeta, Base)):
+    """Mentionee.
```

Comment:
Please change to `Mention`

Reply:
This model Class is meant **information of mentioned user**.
I think that `Mentionee` is better than `Mention`.

https://developers.line.biz/en/news/2021/01/20/messaging-api-update-january-2021/

Reply:
Oh, I was lacking this word.